lib/helpers/url-utils.js

```diff
@@ -61,6 +61,9 @@ function appendQueryObject(url, queryObject) {
         })
         .join(',');
     }
+    if (typeof value === 'object') {
+      value = JSON.stringify(value);
```

Comment:
Maybe we should stringify addlayer the same way so we don't have to change this function at all!